easybuild/easyblocks/n/neuron.py

```diff
@@ -57,6 +59,7 @@ def extra_options():
 
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
+            'enable_tests': [True, "Enable extensive testing in the sanity check step.", CUSTOM],
```

Comment:
@boegel for 2 reasons: 1) I forgot about it; 2) This is normally used to control the `test_step` method. A good followup question is: why are these tests run in the sanity check instead of in the test step? Maybe we should move them out?